app/views/unattended/provisioning_templates/registration/linux_registration_default.erb

```diff
@@ -20,6 +20,23 @@ set -e
 <%= snippet 'puppet_setup' %>
 <% end -%>
 
+<%= snippet 'remote_execution_ssh_keys' %>
+
+<% if host_param_true?('host_registration_insights') -%>
+if [ -f /etc/os-release ] ; then
+  . /etc/os-release
+fi
+
+if [ x$ID = xrhel ]; then
```

Comment:
Since this is a host registration template, we already have the info in the host object. No need to perform this on the actual system.

Reply:
yeah, for a few examples of how to work with @host os information, see  e.g. https://github.com/theforeman/foreman/blob/develop/app/views/unattended/provisioning_templates/provision/kickstart_default.erb#L41-L42

HRT should fully benefit of the Foreman DB infromation